src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -18,21 +17,31 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
             + "the specified keywords (case-insensitive) or tags and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]... t/TAG [MORE_TAGS]\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie t/colleague";
+            + "Parameters: KEYWORD [MORE_KEYWORDS]... t/TAG [MORE_TAGS] pr/PRICE RANGE\n"
+            + "Example: " + COMMAND_WORD + " alice bob charlie t/colleague pr/<100.00";
 
-    private final Predicate<Person> predicate;
+    public static final String PRICE_USAGE = "Valid prices operators are =, <, >, <=, >=. \n"
+            + "Prices must be in 2 decimal. Eg. pr/<100.00";
 
-    public FindCommand(Predicate<Person> predicate) {
+    private final Predicate<Person> predicate;
+    private final String message;
+
+    /**
+     * Creates FindCommand with a predicate an a message describing the query
+     * @param predicate List filter
+     * @param message Message of query
+     */
+    public FindCommand(Predicate<Person> predicate, String message) {
         this.predicate = predicate;
+        this.message = message;
     }
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.updateFilteredPersonList(predicate);
-        return new CommandResult(
-                String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
+        return new CommandResult(""
```

Comment:
Not sure what is the purpose of "" here, but if it is intentional then it's fine i guess